docs/whatsnew-1.10.rst

```diff
@@ -0,0 +1,131 @@
+What's New in Pyramid 1.10
+==========================
+
+This article explains the new features in :app:`Pyramid` version 1.10 as compared to its predecessor, :app:`Pyramid` 1.9. It also documents backwards incompatibilities between the two versions and deprecations added to :app:`Pyramid` 1.10, as well as software dependency changes and notable documentation additions.
+
+.. note::
+
+    This release is the last planned release feature release to support Python 2.7.
```

Comment:
Too many releases.

This release is the last planned feature release to support Python 2.7.

Reply:
This was fixed already you can see it in master.